header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -280,6 +280,8 @@ public function define_row_settings( $row_id ) {
 					'default'           => '{ "mobile": "0", "tablet": "0", "desktop": "0" }',
 				]
 			);
+
+			do_action( 'neve_add_settings_to_hfg_rows', SettingsManager::get_instance(), $row_setting_id, $row_id );
```

Comment:
action should use a prefix of `hfg` rather than `neve` as we plan to bootstrap this as a standalone library. Moreover, previously filter/actions used in this library was using the same pattern.